### ISSUE TITLE
AniDB scraper scrapes series info but not episodes info. 

### ISSUE THREAD

**Issue by DiMag** [tinyMediaManager/tinyMediaManager#406]
Version: 2.9.14-SNAPSHOT - NIGHTLY
Build: 2018-09-25 23:03
OS: Windows 10 10.0
JDK: 1.8.0_162 amd64 Oracle Corporation

__What is the actual behaviour?__
Two anime series, one 37 the other 22 episodes long, were correctly identified and scraped 
with AniDB chosen as metadata source. Then, however, when pressing 'scrape episode metadata', 
nothing happened. Episodes were renamed first as Ep 01 etc then as 1x01 etc, but nothing changed.  
When the scraper source changed to TheTVDb, all episodes (now renamed 1x01) were correctly
scraped.

__What is the expected behaviour?__
AniDB scraper provides episodes metadata as found in anidb.net

__Steps to reproduce:__

__Additional__
Have you attached the logfile from the day it happened?

[tmm_logs.zip](https://github.com/tinyMediaManager/tinyMediaManager/files/2450850/tmm_logs.zip)

**Comment by myron0815** [comment]
Well, you first need to make sure, that all the files are correctly named, so that we can parse them correctly.
See various supported formats: https://kodi.wiki/view/Naming_video_files/TV_shows#Episode_Files
After that, when we have a correct season and episode information, the scraper works...

If i try to update file "Psycho-Pass (2012-2013)/1x10 - Methuselah's Game.mkv", TMM correctly lists it as Season 1, Episode 10 within GUI.

**Comment by michaelothing** [comment]
I have this same problem with all scrappers. They rename the episodes only when they want with the episode name. It works once and then the next time not. Very annoying as I was looking for a replacement for filebot to rename for kodi since Java has so many issues on Arch Linux. This is the only tool I could get to sort of work.

It will scrape once and rename with episodes sometimes and others even for the same series only with the episode and season number which it doesn't always get right. I end up having to manually bulk rename and replace the extra crap, and then mouse select and delete each file if it has database numbers as well in the title because half the time these are interpreted as the episode or season number. I have this on this inconsistent behaviour even on  the same series from the same source as I try to correct tiny's mistakes. 

Renaming is a part of the functionality of this program, no?

**Comment by mlaggner** [comment]
well instead of complaining that tmm should not work (strange that thousands of other users do not have this issues..) provide us EXAMPLES to find the problems

**Comment by DiMag** [comment]
I should have added that this failure to scrape episode names then rename episode files accordingly is attributable not to anything tinyMM does or does not but to the anidb scrapper itself. It appears that ScudLee's modified anidb scrapper (in Kodi repositories) does not add this functionality either.

**Comment by michaelothing** [comment]
DiMag it does inside of Kodi itself I believe. It has been a while since I loaded one up that way. It just doesn't change the actual name of the file. You can set Kodi to rename files manually though is settings.

**Comment by michaelothing** [comment]
mlaggner I don't really know how to reproduce it as for every show I have tried to scrape only 30% of the time will tiny scrape the episode name. You keep harping that files must be named correctly, but, I was under the impression that tiny was also among other things a renaming tool, which is what I am trying to use it for. I get enough info from within kodi itself once an episode or movie is scraped and all these text files just litter up my system.  I found Tiny because Kodi lists it as one of the three filerenamers that work on linux.

**Comment by michaelothing** [comment]
DiMag unless something has changed the version in the kodi repository is the normal AniDB one. Scudlee's AniDB(MOD) has to be downloaded and installed separately as a zip. If you are still using the Kodi repository version I highly recommend downloading Scudlee's repository from here as it is vastly superior: https://forum.kodi.tv/showthread.php?tid=142835 It finds many more titles to include current ones and scrapes episode info too.

**Comment by michaelothing** [comment]
mlaggner the problem I am having is with Bleach as it AniDB has all 366 episodes as season 1. If you scrape with absolute numbering you get the correct NFO files. I know as I tried after changing season and episode number all by hand. If you split it up into 16 seasons as some sources have it and Filebot splits it up, only the first 20 or so episodes get scraped after that for the other 340 episodes AniDB can't scrape the episodes. I have yet to find an easy way to changed all seasons to season one so I can get all 366 info files or rename them in a way that kodi will recognize. It doesn't work in AniDB or ANiDB [MOD] from within Kodi or from TMM.

Have I now explained the problem which I am having. The program is looking for season for a program that there are no seasons for.

**Comment by myron0815** [comment]
Problem is the file nameing.
Kodi, TMM and others usually try to detect the season and episode numbers from various patterns.
It has been established, that 2 numbers for episodes "ought to be enugh".
(History already has us proven wrong ^^)

There is no such thing like "import in absolute, ignore seasons".
YES, while scraping all can handle this, but not on importing.

But since importing is the predecessor for a correct scrape, this needs to be addressed first...

**Comment by michaelothing** [comment]
I suppose the real problem is i am trying to use this program to rename my files. I want them to show up as season one so they will be correctly scraped. i have achieved this by renaming with filebot in 16326  since it can recognize absp'olute numbering and just adds the wrong season number, and then using my standard file managers bulk renaming feature to edit the 1,2,3, - 16 out with s1e. Then using AniDb(mod) from within Kodi all 326 episodes have them all scraped correctly with episode information. It took a little longer than normal, but, using that program i achieved what i was trying to do which was rename and organize my files to begin with and not all the excess stuff that TMM does. I tried TMM because I was.having trouble getting my prefered program running. Now it works and I have made the sugestion here. i am not really interested anymore. Kodi can handle 3 digit episode numbers as long as there is a season number. i realized that from the start as well.
